Cargo.lock

```diff
@@ -4982,14 +5342,41 @@ checksum = "cda74da7e1a664f795bb1f8a87ec406fb89a02522cf6e50620d016add6dbbf5c"
 
 [[package]]
 name = "tokio"
-version = "1.28.2"
+version = "1.16.1"
```

Comment:
Probably need to run around deps and sync some of them back: versions changed from PR creating time?

Reply:
The PR is not that old so that tokio would have 1.16.1. This is the result of some cargo unification. I'll try to reset `Cargo.lock`.

Reply:
Yeah it was only used before as a dev dependency of clippy, so now it was overwritten, since `opt-dist` depends on it.